packages/components/pie-textarea/src/index.ts

```diff
@@ -19,25 +22,48 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
     static shadowRootOptions = { ...LitElement.shadowRootOptions, delegatesFocus: true };
 
     @property({ type: Boolean, reflect: true })
-    public disabled?: TextareaProps['disabled'];
+    public disabled = defaultProps.disabled;
 
     @property({ type: String })
     @validPropertyValues(componentSelector, sizes, defaultProps.size)
-    public size?: TextareaProps['size'] = defaultProps.size;
+    public size = defaultProps.size;
+
+    @property({ type: String })
+    @validPropertyValues(componentSelector, resizeModes, defaultProps.resize)
+    public resize = defaultProps.resize;
+
+    @query('textarea')
+    private _textarea!: HTMLTextAreaElement;
+
+    private handleResize () {
```

Comment:
Should this be debounced / limited?

Reply:
Yeah it's an interesting question. I don't think debounce would work very well because it can feel unresponsive as you're typing. Throttling might work if we get the timing right. Maybe something like 100-200ms.

Reply:
Yup throttling is the one i always mix those up. 👍